public/js/controllers/dataController.js

```diff
@@ -12,9 +13,32 @@ controllers.controller("dataController", ["$scope", "$http", "dataAnalyzer",
         url: "/api/pull"
 
       }).success(function(data) {
-        // Sends data to the data analyzer service for formatting
+        console.log(data.stats);
+
+        // Extract comments
         dataAnalyzer.extractComments($scope, data);
-        dataAnalyzer.extractSubreddit($scope, data);
+
+        // Extract subreddit data
+        subData = dataAnalyzer.chartData($scope, data, "subreddit");
+        $scope.postsBySub = subData.posts;
+        $scope.scoreBySub = subData.score;
+        $scope.gildedBySub = subData.gilded;
+
+        // Extract domain data
+        domainData = dataAnalyzer.chartData($scope, data, "domain");
+        $scope.postsByDomain = domainData.posts;
+        $scope.scoreByDomain = domainData.score;
+        $scope.gildedByDomain = domainData.gilded;
+
+        // Extract author data
+        authorData = dataAnalyzer.chartData($scope, data, "author");
+        $scope.postsByAuthor = authorData.posts;
+        $scope.scoreByAuthor = authorData.score;
+        $scope.gildedByAuthor = authorData.gilded;
+
+        console.log(subData);
```

Comment:
Not now, but at some point we should grep and clean up some of our console.log()'s